tests/func/test_data_cloud.py

```diff
@@ -69,6 +70,12 @@ def _should_test_aws():
     return False
 
 
+def _should_test_gdrive():
+    if os.getenv("DVC_TEST_GDRIVE") == "true":
```

Comment:
So this test is disabled by default? Why though?

Reply:
Thank you for taking a look,
I've overlooked somehow that there is the test to send/receive real remote queries.
Seems, need to add back some functionality from original PR to decode credentials and make authorization configurable.

Reply:
in previous PR that test also was skipped https://travis-ci.com/iterative/dvc/jobs/220648788#L961 , but seems only because TravisCI was not configured with environment variables to decode oauth2 tokens https://github.com/iterative/dvc/pull/2040/files#diff-07aeba083b9ed621019d82dec72ae4baR7
I hope that it should be possible to store whole oauth2 token data just directly in secure travis ci env variable. And it will not be needed to decode previously encoded file.

Reply:
@MaxRis Let me know if you need any help with that. I understand that this PR is from your fork, so travis won't be able to decode those env vars, but it is fine as long as you can confirm that it works for you with those credentials. It is totally fine if we need to make small adjustments to that after the PR is merged.

Reply:
@efiop so far I've tried to read all oauth2 credentials for PyDrive just from environment variable and locally it works fine (except, that implementation for `list_cache_path` is missed to pass test_data_cloud test successfully). It looks like:
```
    if os.getenv("PYDRIVE_USER_CREDENTIALS_FILE_CONTENT"):
        with open("credentials.json", "w") as credentials_file:
           credentials_file.write(os.getenv("PYDRIVE_USER_CREDENTIALS_FILE_CONTENT"))
```